src/Test/Unit/StaticContent/Deploy/OptionTest.php

```diff
@@ -13,8 +13,8 @@
 use Magento\MagentoCloud\Package\MagentoVersion;
 use Magento\MagentoCloud\StaticContent\Deploy\Option;
 use Magento\MagentoCloud\StaticContent\ThreadCountOptimizer;
+use PHPUnit\Framework\MockObject\MockObject;
```

Comment:
Why did you do these changes in this pr? It's not related to the task, please create a separate task for such changes and change them all at once.
In this Pr changed 147 files,  but maybe 20 related to the task, it's difficult to review.

Reply:
Fixed